diff --git a/timezone/zdump.c b/timezone/zdump.c
index 065d5ab..e7ffcf5 100644
--- a/timezone/zdump.c
+++ b/timezone/zdump.c
@@ -38,6 +38,9 @@
 #include <string.h>
 #include <time.h>
 
+#ifdef __GNUC__
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
+#endif
 
 #ifndef TYPE_BIT
 #define TYPE_BIT(type)	(CHAR_BIT * (int) sizeof (type))
diff --git a/timezone/zic.c b/timezone/zic.c
index 342fabc..75bbf4e 100644
--- a/timezone/zic.c
+++ b/timezone/zic.c
@@ -49,6 +49,9 @@
 #include <time.h>
 #include <unistd.h>
 
+#ifdef __GNUC__
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
+#endif
 
 #ifndef ZIC_MAX_ABBR_LEN_WO_WARN
 #define ZIC_MAX_ABBR_LEN_WO_WARN	6
